docs/en/engines/table-engines/integrations/kafka.md

```diff
@@ -125,6 +125,10 @@ Groups are flexible and synced on the cluster. For instance, if you have 10 topi
 2.  Create a table with the desired structure.
 3.  Create a materialized view that converts data from the engine and puts it into a previously created table.
 
+:::info
+Kafka engine doesn't support `DEFAULT/EPHEMERAL/ALIAS` section for columns. If you need any of these sections, they can be added at `MATERIALIZED VIEW` level.
```

Comment:
Did you mean "The kafka Kafka table engine don't support columns with default value of type `DEFAULT`, `EPHEMERAL` or `ALIAS`"?

Please also link the keywords to the docs [here](https://clickhouse.com/docs/en/sql-reference/statements/create/table/#default-values).